contracts/0.8.9/oracle/ValidatorsExitBusOracle.sol

```diff
@@ -147,11 +147,15 @@ contract ValidatorsExitBusOracle is BaseOracle, PausableUntil {
         ///
 
         /// @dev Total number of validator exit requests in this report. Must not be greater
-        /// than limit checked in OracleReportSanityChecker.checkExitBusOracleReport
+        /// than limit checked in OracleReportSanityChecker.checkExitBusOracleReport.
+        ///
+        /// Cannot be zero: in the case there's no newly exited or stuck validators at the moment
```

Comment:
Didn't get the meaning of this comment.

AFAIK, we don't report exited and stuck validators from the ValidatorExitButOracle (using the accounting one instead).

Reply:
F, you're right. Will fix.